velero/backup/common-service/label-common-service.sh

```diff
@@ -0,0 +1,184 @@
+        
+        # Label the entitlement key
+        ${OC} label secret ibm-entitlement-key foundationservices.cloudpak.ibm.com=entitlementkey -n "$namespace" --overwrite=true 2>/dev/null
+        
+        # Label the OperandRequest
+        operand_requests=$(${OC} get operandrequest -n "$namespace" -o custom-columns=NAME:.metadata.name --no-headers)
+        # Loop through each OperandRequest name
+        while IFS= read -r operand_request; do
+            ${OC} label operandrequests $operand_request foundationservices.cloudpak.ibm.com=operand -n "namespace" --overwrite=true 2>/dev/null
+        done <<< "$operand_requests"
+    done <<< "$namespaces"
+
+    ${OC} label secret ibm-entitlement-key foundationservices.cloudpak.ibm.com=entitlementkey -n $DEFAULT_SOURCE_NS --overwrite=true 2>/dev/null
+    ${OC} label secret pull-secret -n openshift-config foundationservices.cloudpak.ibm.com=pull-secret --overwrite=true 2>/dev/null
+
+}
+
+function label_configmap() {
+    
+    title "Start to label the ConfigMaps... "
+    ${OC} label configmap common-service-maps foundationservices.cloudpak.ibm.com=configmap -n kube-public --overwrite=true 2>/dev/null
+    ${OC} label configmap cs-onprem-tenant-config foundationservices.cloudpak.ibm.com=configmap -n $SERVICES_NS --overwrite=true 2>/dev/null
+    ${OC} label configmap platform-auth-idp foundationservices.cloudpak.ibm.com=configmap -n $SERVICES_NS --overwrite=true 2>/dev/null
+}
+
+function label_subscription() {
+
+    title "Start to label the Subscriptions... "
+    local cs_pm="ibm-common-service-operator"
+    local cm_pm="ibm-cert-manager-operator"
+    local lis_pm="ibm-licensing-operator-app"
+    local lsr_pm="ibm-license-service-reporter-operator"
+    
+    ${OC} label subscriptions.operators.coreos.com $cs_pm foundationservices.cloudpak.ibm.com=subscription -n $OPERATOR_NS --overwrite=true 2>/dev/null
+    ${OC} label subscriptions.operators.coreos.com $cm_pm foundationservices.cloudpak.ibm.com=subscription -n $CERT_MANAGER_NAMESPACE --overwrite=true 2>/dev/null
+    ${OC} label subscriptions.operators.coreos.com $lis_pm foundationservices.cloudpak.ibm.com=subscription -n $LICENSING_NAMESPACE --overwrite=true 2>/dev/null
+    ${OC} label subscriptions.operators.coreos.com $lsr_pm foundationservices.cloudpak.ibm.com=subscription -n $LSR_NAMESPACE --overwrite=true 2>/dev/null
+}
+
+function label_cs(){
+    
+    title "Start to label the CommonService CR... "
+    ${OC} label commonservices common-service foundationservices.cloudpak.ibm.com=commonservice -n $OPERATOR_NS --overwrite=true 2>/dev/null
+    ${OC} label customresourcedefinition commonservices.operator.ibm.com foundationservices.cloudpak.ibm.com=crd --overwrite=true 2>/dev/null
+    ${OC} label operandconfig common-service foundationservices.cloudpak.ibm.com=operand -n $SERVICES_NS --overwrite=true 2>/dev/null
+}
+
+main $*
```

Comment:
I think it would be a good idea to include some kind of success statement at the end to signify the script exited successfully as expected. As it stands now, the script just ends and I had to go back and make sure nothing else was supposed to run.